### PR TITLE
SRVKP-6442 Enhance default credentials setup of the cache stepaction

### PR DESCRIPTION
**Pull request by pramodbindal**

### CODE REVIEW

File: tekton/cache-fetch.yaml [hunk 1]
```diff
@@ -38,6 +38,11 @@ spec:
         The path where to find the google credentials. If left empty, it is ignored.
       type: string
       default: ""
+    - name: docker-config
+      description: |
+        The path where to find the docker config.json file. If left empty, it is ignored.
+      type: string
+      default: "/workspace/docker"
```
Comment (vdemeester) [inline tekton/cache-fetch.yaml:45]: I wonder if the name should be *just* `docker` for the workspace or something else.

Comment (pramodbindal) [reply tekton/cache-fetch.yaml:45]: > I wonder if the name should be _just_ `docker` for the workspace or something else.

what about .docker ?

Comment (vdemeester) [reply tekton/cache-fetch.yaml:45]: I was more thinking of `docker-config` or something that makes it a bit more clear what we are "waiting" for.

File: tekton/cache-upload.yaml [hunk 2]
```diff
@@ -47,6 +47,11 @@ spec:
       description: |
         The path where to find the google credentials. If left empty, it is ignored.
       type: string
+      default: "/workspace/docker"
```
Comment (vdemeester) [inline tekton/cache-upload.yaml:50]: btw, this shouldn't be on `googleCredentialsPath` param though (but on `docker-config`).

Comment (pramodbindal) [reply tekton/cache-upload.yaml]: Fixed

**Review by vdemeester — APPROVED** [review]

**Pull request merged**